data/sql/updates/pending_db_world/rev_1619259290727738768.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619259290727738768');
+
+SET @INCENDIA_AGAVE_ENTRY := 175928;
+
+DELETE FROM `gameobject` WHERE (`id` = @INCENDIA_AGAVE_ENTRY) AND (`guid` IN (2135276, 2135277, 2135278, 2135279));
```

Comment:
These are very high GUIDs...

Use 

6024,
6025,
6026,
6027,

Instead. This way we can fill gaps in the DB.

You can use tools like https://github.com/Kitzunu/Unused-Guid-Search/tree/master to find open entries